src/Interpreters/QueryLog.h

```diff
@@ -86,6 +88,9 @@ struct QueryLogElement
     std::shared_ptr<ProfileEvents::Counters::Snapshot> profile_counters;
     std::shared_ptr<Settings> query_settings;
 
+    std::vector<String> row_policies_names;
+    std::vector<RowPolicyOrderType> row_policies_order_types;
```

Comment:
> along with the final combined row policy filter.  

I was afraid (a bit) that exposing the filter itself may not be desired sometimes

```
WHERE client = 'bad_guy'
```

or can expose some internals 

```
WHERE team IN ('bookeeping', 'marketing') -- i have no access to the row but it points me the direction of the attack
```

But I'm not sure if it's a real problem (names of row policies themself can also have similar problems), so all options are acceptable for me.

Reply:
So that permissive / restrictive can help a bit there if there is now filter visible.

Reply:
Good point! We could write UUIDs of row policies instead of their names. Then it will be necessary to have the rights to read from `system.row_policies` to find information about actual row policies used.

Reply:
> We could write UUIDs of row policies instead of their names. 

Also thought about the same. That is the safest option. No so handy, but I'm ok with that and it solves the main problem

Reply:
I think we shall better return both names and UUIDs, for both human and machine convenience. If someone doesn't want to expose names she can obfuscate them.

Reply:
@vitlibar Are you fine with that?

Reply:
I discussed that with our team.

Let's step back and make the query log show only the names of row policies, not UUIDs. Because it's more convenient to analyze.

About the concern that those names can give a hint for an attacker it seems that was a bit far-fetched. First it's possible not to give access to some columns of the query log for some users. And second the names of row policies unlikely will contain secret information (they're like the names of columns or tables).

Reply:
Done.